docs/articles_en/documentation/openvino_legacy_features.rst

```diff
@@ -90,14 +89,11 @@ offering.
 |   `Check the NNCF GitHub project, including documentation <https://github.com/openvinotoolkit/nncf>`__
 
 
-| **Old Inference API 1.0**
+| **Inference API 1.0**
```

Comment:
can we need to completely remove mentioning Inference Engine API 1.0 ? as it's completely removed in 2024.0

Reply:
It will be mentioned only in the "legacy features" article, at the bottom, with all other discontinued technologies. The idea is to keep some info on all discontinued solutions, at least for as long as they exist in LTS versions. It will be a sort of a log of old stuff, R.I.P components.

Reply:
But here we are talking about 2024.0 which is NOT LTS :)
Personally, I prefer to remove legacy at all and not to keep it for history, but up to you.

Reply:
Yeah, that's true :) I'd prefer to keep it, PMs like it ;) This article will just let people know what is no longer in the current version but may be found in the two LTS versions that are still maintained (with links). Once we have release notes done properly and not on IDZ, we can clean this article from discontinued stuff.